### PR TITLE
make sure syndesis-operator CLI is running with the right context file

### PR DESCRIPTION
**Pull request by wei-lee**
## Additional Information
https://issues.redhat.com/browse/INTLY-4508

### CODE REVIEW

File: inventories/group_vars/osd/osd.yml [hunk 1]
```diff
@@ -84,3 +84,5 @@ integreatly_operator: False
 
 # Increasing prometheus volume size for prod environments to 50gb
 monitoring_prometheus_storage_request: 50Gi
+
```
Comment (pmccarthy) [inline inventories/group_vars/osd/osd.yml]: @wei-lee Really small detail but would you mind adding a comment explaining why this var is set? All other vars in this file have a comment associated with them

File: inventories/group_vars/poc/poc.yml [hunk 2]
```diff
@@ -48,3 +48,5 @@ ups: True
 
 # Use Integreatly Operator for Installation
 integreatly_operator: False
+
```
Comment (pmccarthy) [inline inventories/group_vars/poc/poc.yml]: As per previous comment

**Comment by pmccarthy** [comment]
Changes look good. Will wait for @damienomurchu to validate before approving

**Comment by damienomurchu** [comment]
integreatly-install from these changes just running on a POC cluster now (https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/openshift-cluster-integreatly-install/605/console)

**Comment by wei-lee** [comment]
@damienomurchu this should fix the other issue you were having when provision POC clusters. You can probably just run the same installation job again and it should just continue.

**Comment by wei-lee** [comment]
@pmccarthy do you mind approve this PR? I will then cherry-pick it to master. Thanks.

**Comment by damienomurchu** [comment]
@pmccarthy can you approve? I don't have permission to approve

**Pull request merged**